nightly_tests/run.sh

```diff
@@ -256,11 +261,11 @@ fi
 #
 # Parse and print out CloudFormation events
 #
-cat cloudformation_events.txt |sed 's/\s*},*//g' |sed 's/\s*{//g' |sed 's/\s*\]//' |sed 's/\\"//g' |sed 's/"//g' |sed 's/\\n//g' |sed 's/\\/-/g' |sed 's./.-.g' |sed 's.\\.-.g' |sed 's/\[//g' |sed 's/\]//g' |sed 's/  */ /g' |sed 's/%//g' |grep -v StackName |grep -v StackId |grep -v PhysicalResourceId > CF_EVENTS.txt
-EVENTS=$(cat CF_EVENTS.txt |grep -v ResourceProperties)
-echo "$EVENTS" > CF_EVENTS.txt
-cat CF_EVENTS.txt
-CF_EVENTS=$(cat CF_EVENTS.txt)
+# cat cloudformation_events.txt |sed 's/\s*},*//g' |sed 's/\s*{//g' |sed 's/\s*\]//' |sed 's/\\"//g' |sed 's/"//g' |sed 's/\\n//g' |sed 's/\\/-/g' |sed 's./.-.g' |sed 's.\\.-.g' |sed 's/\[//g' |sed 's/\]//g' |sed 's/  */ /g' |sed 's/%//g' |grep -v StackName |grep -v StackId |grep -v PhysicalResourceId > CF_EVENTS.txt
```

Comment:
just get rid of these lines, if there is no value in keeping them

Reply:
CF_ENVENTS is used by the slack curl command, CF_EVENTS holds the output of the smoke_test.sh. There is probably a more elegant way of getting the output to the curl slack command. I was reusing that variable since it has been deprecated. Will look into this.